### PR TITLE
fix: [CDS-58334]: loading state for service list when service is expression

### PR DESCRIPTION
**Pull request by AshwinBhatkal-Harness**
### Summary

The Spinner in service tab is persistent when Deploy to multiple services is toggled on when service is expression. This has been fixed now

#### Screenshots

https://user-images.githubusercontent.com/96711906/232865423-fe0130a6-c630-4d5c-abdc-569550e76748.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/75-cd/strings/strings.en.yaml [hunk 1]
```diff
@@ -310,13 +311,14 @@ pipelineSteps:
...
-    singleServicesTitleText: Deploy to single service?
-    singleServicesConfirmationText: Are you sure you want to switch to deploying to a single services?. If you choose to proceed, your existing services will be lost.
-    deleteServiceFromListTitleText: Remove service from list?
-    deleteServiceFromListText: Are you sure you want to remove {{name}} service from the list?
-    viewServiceInputs: View service inputs
-    hideServiceInputs: Hide service inputs
+    multiServicesConfirmationText: Are you sure you want to switch to deploying multiple Services?. If you choose to proceed, your existing Service will be migrated automatically.
+    multiServicesClearConfirmationText: Are you sure you want to switch to deploying multiple Services? If you choose to proceed, your existing Service will be lost.
+    singleServicesTitleText: Deploy to single Service?
+    singleServicesConfirmationText: Are you sure you want to switch to deploying to a single Services?. If you choose to proceed, your existing Services will be lost.
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/strings/strings.en.yaml:317]:
```suggestion
    singleServicesConfirmationText: Are you sure you want to switch to deploying to a single Service?. If you choose to proceed, your existing Services will be lost.
```

**Review by vikrant-harness — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx [hunk 2]
```diff
@@ -373,7 +378,7 @@ export default function DeployServiceEntityWidget({
     }
   }
 
-  function updateValuesInFomikAndPropogate(values: FormState): void {
+  function updateValuesInFormikAndPropogate(values: FormState): void {
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:381]: let's fix the other typo too, propogate -> propagate

**Review by rameshx — APPROVED** [review]

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx (hunk 2 above)
Comment (AshwinBhatkal-Harness) [reply src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx:381]: Ahaa. Will fix it in another PR. Thanks

Comment (AshwinBhatkal-Harness) [reply src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/DeployServiceEntityWidget.tsx]: Fixed

**Review by vikrant-harness — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Pull request merged**